worker/docker.go

```diff
@@ -57,20 +57,57 @@ var containerHostPorts = map[string]string{
 }
 
 // Mapping for per pipeline container images.
+var defaultBaseImage = "livepeer/ai-runner:latest"
 var pipelineToImage = map[string]string{
 	"segment-anything-2": "livepeer/ai-runner:segment-anything-2",
 	"text-to-speech":     "livepeer/ai-runner:text-to-speech",
 	"audio-to-text":      "livepeer/ai-runner:audio-to-text",
 	"llm":                "livepeer/ai-runner:llm",
 }
-
 var livePipelineToImage = map[string]string{
 	"streamdiffusion":    "livepeer/ai-runner:live-app-streamdiffusion",
 	"comfyui":            "livepeer/ai-runner:live-app-comfyui",
 	"segment_anything_2": "livepeer/ai-runner:live-app-segment_anything_2",
 	"noop":               "livepeer/ai-runner:live-app-noop",
 }
 
+// overridePipelineImages updates base and pipeline images with the provided overrides.
+func overridePipelineImages(imageOverrides string) error {
+	if imageOverrides == "" {
+		return fmt.Errorf("empty string is not a valid image override")
+	}
+
+	// Handle JSON format for multiple pipeline images.
+	var imageMap map[string]string
+	if err := json.Unmarshal([]byte(imageOverrides), &imageMap); err == nil {
+		for pipeline, image := range imageMap {
+			if pipeline == "base" {
+				defaultBaseImage = image
+				continue
+			}
+
+			// Check and update the pipeline images.
+			if _, exists := pipelineToImage[pipeline]; exists {
+				pipelineToImage[pipeline] = image
+			} else if _, exists := livePipelineToImage[pipeline]; exists {
+				livePipelineToImage[pipeline] = image
```

Comment:
I think the override JSON format could be improved to avoid multiple conflicts here. We have 3 different identifiers as keys on the same map: 
- the special `base` image name which is used as the default here
- the batch pipeline image names (which comes from the `pipeline_id`)
- the live pipeline image names (which comes from the `model_id`)

All of these values could conflict with each other. I think we could have a layered JSON and avoid this, like:
```
type ImageOverrides struct {
    Default string
    Batch map[string]string
    Live map[string]string
}
```
Then we access each of them independently. WDYT?

Reply:
Love it 😍. I did briefly have just this but then removed it again to keep changes minimal.